bin/handler-twiliosms.rb

```diff
@@ -37,13 +37,8 @@ def handle
     raise 'Please define a valid set of SMS recipients to use this handler' if candidates.nil? || candidates.empty?
 
     recipients = []
-    candidates.each do |mobile, candidate|
```

Comment:
I dont think this is the same event filtering that is being logged about, I think you would need to update [this](https://github.com/sensu-plugins/sensu-plugins-twilio/blob/1.0.0/sensu-plugins-twilio.gemspec#L32) much like I did [here](https://github.com/sensu-plugins/sensu-plugins-docker/pull/57). While I do think that this type of filtering should probably come out and be put into a filter I think this should stay for now and remove just the filtering referenced in the article and adress removing this filtering in a separate PR and we can ask if anyone is using the functionality before deciding that we need to axe it.

Reply:
well you have not bumped the version of the dependency in the `.gemspec` to pull in `2.0` so lets try pulling that in.

Reply:
I bumped the .gemspec and removed all my other changes at this point. I'm still trying to figure out how to write tests for the Twilio part of the plugin.